Bundle/OdmMetadata.php

```diff
@@ -133,7 +133,7 @@ public function getDocumentNames()
     /**
      * @return array|\Iterator
```

Comment:
Either remove the array from PHPdoc or remove the return type

Reply:
Why would I do that? The PHPdoc is more precise, and I can't express that with a return type.